src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java

```diff
@@ -53,4 +55,11 @@ private LocalDateTime createAtToCreateDate(CmcClaim claim) {
 
         return createdDate;
     }
+
+    private LocalDateTime setIntentionToProceedDeadline(CmcClaim claim) {
```

Comment:
this is not a good method in my opinion. it states that it sets something but instead it is getting intention to proceed deadline from cmc. perhaps this method should be renamed to get and should reside in cmc claim. then you would call cmcClaim.getIntentonToProceedDeadline(); in the builder

Reply:
changed to cmcClaim.getIntentonToProceedDeadline() , removed setIntentionToProceedDeadline method